duckduckgo/iOS
# Subscriptions - 21. Manage Billing options to third parties

Task/Issue URL: https://app.asana.com/0/0/1206397634707138/f

**Description**:
- Allows the user to display third party billing information (Google/Stripe)



**Steps to test this PR**:
1.  Get a Stripe subscription on macOS
2. Add Email
3. Restore on iOS
4. Go to Subscription Settings (iOS)
5. Tap Change Plan or Billing
6. Observe you're taken to the Stripe billing page

⚠️  If DDG is not the default browser, Safari or the default browser will be used.  This is expected to comply with Apple Guidelines

For Google, we just have the following static View:  (No need to further testing)

<img width="412" alt="Screenshot 2024-03-12 at 1 29 36 AM" src="https://github.com/duckduckgo/iOS/assets/1156669/f917b12e-b2ff-43a5-96f0-2e4e2776f37a">





---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
